src/components/common/NewModalDialog/index.tsx

```diff
@@ -85,7 +84,7 @@ const NewModalDialog = ({
         <span style={{ flex: 1 }} />
         <IconButton
           aria-label="close"
-          onClick={() => setTxFlow(undefined)}
+          onClick={(e) => onClose?.(e, 'backdropClick')}
```

Comment:
Do we have to call `onClose` with e and backdropClick here even if its not being used?

Reply:
Yeah, that's the signature of that prop inherited from the MUI Dialog.